component_library.csv

```diff
@@ -9,10 +9,16 @@ CAPP-007,Digi-Key,587-2909-1-ND,CAP CER 2.2UF 25V 10% X5R 0603,,99,9/18/17,
 CAPP-008,Digi-Key,732-7775-1-ND,CAP CER 15PF 25V C0G/NP0 0603,,100,9/18/17,
 CAPP-009,Digi-Key,732-7773-1-ND,CAP CER 6.8PF 25V C0G/NP0 0603,,100,9/18/17,
 CAPP-010,Digi-Key,490-1512-1-ND,CAP CER 10000PF 50V X7R 0603,,100,9/18/17,
+CAPP-011,Digi-Key,399-1153-1-ND,CAP CER 3300PF 50V X7R 0805,,0,2/8/2018,
+CAPP-012,Digi-Key,490-12389-1-ND,CAP CER 22UF 25V X5R 0805,,0,2/8/2018,
+CAPP-013,Digi-Key,445-14433-1-ND,CAP CER 6.8UF 35V X5R 0805,,0,6/11/2018,
+CAPP-014,Digi-Key,478-5778-1-ND,CAP CER 0.1UF 50V X7R 0603,AVX Corporation,0,18-06-2018,

```

Comment:
22uF and 0.1uF capacitors already exist, idk if there was a specific reason you added CAPP-012 and CAPP-014 but it looks like you never use CAPP-012 anyways.